holoviews/plotting/bokeh/element.py

```diff
@@ -1983,14 +1992,15 @@ def _get_cmapper_opts(self, low, high, factors, colors):
                                       "Note that the option cnorm='eq_hist' requires "
                                       "bokeh 2.2.3 or higher.")
                 colormapper = EqHistColorMapper
+                if bokeh_version >= LooseVersion('2.4.3'):
```

Comment:
```suggestion
                if bokeh_version > LooseVersion('2.4.2'):
```

At least for testing, this guard allow Bokeh dev releases like 2.4.3rc1, so it can't demand 2.4.3 or later.